notebooks/samples/explanations/tf2/ai-explanations-tabular.ipynb

```diff
@@ -1,1483 +1,1432 @@
 {
```

Comment:
<p>Replace first paragraph in this section with:</p><p> </p><p>In order to deploy this model to AI Explanations, you need to create an explanation_metadata.json file with information about your model inputs, outputs, and baseline. You can use the&nbsp;<a href="https://pypi.org/project/explainable-ai-sdk/" target="_blank">Explainable AI SDK</a>&nbsp;to generate most of the fields.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>